app/com/linkedin/drelephant/analysis/Severity.java

```diff
@@ -23,20 +23,23 @@
  * The severities that you can use
  */
 public enum Severity {
+  @EnumValue("5")
```

Comment:
I have updated the mysql evolution files. We just have to increment the current severity values in the db before deploying Dr. Elpehant.